www_src/components/action-menu/action-menu.less

```diff
@@ -0,0 +1,81 @@
+/*********************************************************
+* Action Menu
+*/
+.action-menu {
+  @_bottomGutter: 10px;
+
+  @_primaryBg: #69A0FC;
+  @_primaryText: #FFF;
+  @_secondaryBg: #FFF;
+  @_secondaryText: #718095;
```

Comment:
I'm thinking we're gonna probably need a colors.less and some kind of standardized palette soon...